.github/workflows/mingw-w64.yml

```diff
@@ -0,0 +1,65 @@
+name: mingw-w64
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    name: Build
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        with-patch:
+          - true
+          - false
+    env:
+      MINGW_PACKAGE_PREFIX: mingw-w64-ucrt-x86_64
+      MSYSTEM: UCRT64
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: dependencies
+        shell: c:\msys64\usr\bin\bash.exe -e -l {0}
```

Comment:
`-l` (login shell) is important here.

If we don't have `-l`, mingw-w64 build environment isn't prepared.